src/components/MoreInfoModal/MoreInfoModal.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/no-array-index-key */
+import 'react-slideshow-image/dist/styles.css';
+
+import CloseIcon from '@mui/icons-material/Close';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { Paper } from '@mui/material';
+import Button from '@mui/material/Button';
+import Modal from '@mui/material/Modal';
+import Typography from '@mui/material/Typography';
+import * as React from 'react';
+import { Slide } from 'react-slideshow-image';
+import photosHouse from 'src/assets/images/House.png';
+import photosMap from 'src/assets/images/Map.png';
+import photosNavLogo from 'src/assets/images/NavLogo.png';
+import photosNoPhoto from 'src/assets/images/nophoto.png';
+
+import FacilitiesList from './FacilitiesList';
+import styles from './MoreInfoModal.module.scss';
+import PropertyInformationList from './PropertyInformationList';
+
+const style = {
+  p: 2,
+  pb: 4,
+};
+
+const houseImages = [photosHouse, photosNavLogo, photosMap, photosNoPhoto];
+
+type HouseInfo = {
+  url: string;
+  caption: string;
+}[];
+
+// Create array with house pohotos for modal
+const slideImages: HouseInfo = [];
+
+houseImages.forEach((houseImage, index) => {
+  slideImages.push({
+    url: houseImage,
+    caption: `Photo ${index}`,
+  });
+});
+
+const slideProperties = {
+  autoplay: false,
+};
+
+const moreInfo = {
+  address: '420 Baker St, London',
+  price: '$299,999',
+};
+
+const propertyInformation = {
+  propertyType: 'House',
+  area: '1200',
+  yearBuilt: '2000',
+  floor: '1',
+  floorsInBuilding: 'House',
+  roomsNumber: '5',
+  bathroomNumber: '2',
+  heating: 'Radiators',
+};
+
+export default function BasicModal() {
```

Comment:
The name is slightly confusing as it differs from the name of this file.

Reply:
Changed ;)